src/app/queue/repositories/mysql.go

```diff
@@ -0,0 +1,84 @@
+package queue_repositories
+
+import (
+	"clinic-api/src/app/queue"
+	"errors"
+	"fmt"
+
+	"github.com/google/uuid"
+	"gorm.io/gorm"
+)
+
+type repository struct {
+	DB *gorm.DB
+}
+
+// InsertData implements queue.Repositories
+func (repo *repository) InsertData(data queue.Domain) (string, error) {
+	var queueNumber int
+	if err := repo.DB.Table("queues").Select("COALESCE(MAX(daily_queue_number), 0)").
+		Where("daily_queue_date = CURDATE() AND polyclinic_id = ?", data.PolyclinicID).
+		Find(&queueNumber).Error; err != nil {
+		return uuid.Nil.String(), err
+	}
+	queueNumber++
```

Comment:
Prefer make this procedure in a new method, and process the number of queue increments on service.

Reply:
done